client/client.go

```diff
@@ -41,7 +43,11 @@ func (c *Client) RegisterTaskDefinition(task, image, tag *string) (string, error
 	}
 	input := &ecs.RegisterTaskDefinitionInput{
 		Family:               task,
+		TaskRoleArn:          taskDef.TaskRoleArn,
```

Comment:
Here's the bug fix.  Along with TaskRoleArn, also carry over NetworkMode, Volumes, and PlacementConstraints